pkg/operator/controller/ingress/metrics.go

```diff
@@ -68,6 +74,23 @@ func DeleteIngressControllerConditionsMetric(ic *operatorv1.IngressController) {
 	}
 }
 
+func DeleteActiveNLBMetrics(ic *operatorv1.IngressController) {
+	activeNLBs.DeleteLabelValues(ic.Name)
+}
+
+func SetIngressControllerNLBMetric(ci *operatorv1.IngressController) {
+	labelVal := 0
+	if ci.Status.EndpointPublishingStrategy.LoadBalancer != nil &&
```

Comment:
ci.Status.EndpointPublishingStrategy should be checked for non-nil, as it is an optional field.  All of these `&&` statements can be put in one `if` clause, there is no need for two `if` clauses.